lib/src/delegates/asset_picker_text_delegate.dart

```diff
@@ -844,3 +845,100 @@ class FrenchAssetPickerTextDelegate extends AssetPickerTextDelegate {
   @override
   String get sUnitAssetCountLabel => 'quantité';
 }
+
+/// Added Turkish Language Support
+///
+/// @date 20220601
+/// @author Yusuf Gündoğdu
+/// @author Cevheri
```

Comment:
We don't use the author field here. Please refer to other delegates to see how to write comments here.